src/data/roadmapData.json

```diff
@@ -238,6 +238,70 @@
           ]
         }
       ]
+    },
+    {
+      "Programming": [
+        {
+          "name": "Javascript",
+          "data": [
+            {
+              "description": "JavaScript is the world's most popular programming language. It started out as purely browser scripting language, after the release of NodeJS which allowed for JS execution outside the browser the language blew up. Today, you can use JavaScript to write all kinds of applications, including browser, server, mobile, and desktop applications.",
+              "url": "https://launchschool.com/books/javascript/read/introduction#briefhistory",
+              "urlLabel": "A brief Javascript history"
+            },
+            {
+              "description": "In this video course you will go through all the Javascript fundamentals.",
+              "url": "https://www.youtube.com/watch?v=2Ji-clqUYnA",
+              "urlLabel": "Javascript fundamentals video"
+            },
```

Comment:
Feel free to add more examples here - added a course that I mostly used and think is worth mentioning (also it is free 😄 )